### PR TITLE
Pod, namespace, node, allKeys lists in (debug) API

### PR DESCRIPTION
**Pull request by mwielgus**

### CODE REVIEW

File: metrics/api/v1/model_handlers.go [hunk 1]
```diff
@@ -50,7 +50,6 @@ func (a *Api) RegisterModel(container *restful.Container) {
 		Consumes("*/*").
 		Produces(restful.MIME_JSON)
 
-	// The /metrics/ endpoint returns a list of all available metrics for the Cluster entity of the model.
```
Comment (piosz) [inline metrics/api/v1/model_handlers.go]: Why are you removing this comment?

File: metrics/api/v1/model_handlers.go [hunk 2]
```diff
@@ -67,6 +66,12 @@ func (a *Api) RegisterModel(container *restful.Container) {
 		Writes(types.MetricResult{}))
 
 	// The /nodes/{node-name}/metrics endpoint returns a list of all available metrics for a Node entity.
+	ws.Route(ws.GET("/nodes/").
```
Comment (piosz) [inline metrics/api/v1/model_handlers.go]: The comment above in wrong. Please fix.

File: metrics/api/v1/model_handlers.go [hunk 3]
```diff
@@ -86,6 +91,12 @@ func (a *Api) RegisterModel(container *restful.Container) {
 		Writes(types.MetricResult{}))
 
 	if a.runningInKubernetes {
+
```
Comment (piosz) [inline metrics/api/v1/model_handlers.go]: Remove empty line

**Comment by piosz** [comment]
LGTM, just a few nits.

File: metrics/api/v1/model_handlers.go (hunk 1 above)
Comment (mwielgus) [inline metrics/api/v1/model_handlers.go]: Actually I planned to remove all comments (they are duplicated in doc, couple lines below).

Comment (mwielgus) [inline metrics/api/v1/model_handlers.go]: Next PR.

File: metrics/api/v1/model_handlers.go (hunk 2 above)
Comment (mwielgus) [inline metrics/api/v1/model_handlers.go]: Done.

File: metrics/api/v1/model_handlers.go (hunk 3 above)
Comment (mwielgus) [inline metrics/api/v1/model_handlers.go]: Ack.

**Pull request merged**